### ISSUE TITLE
about the accuracy compared with the original deep_sort code

### ISSUE THREAD

**Issue by liangzimei** [Qidian213/deep_sort_yolov3#32]
Hi , i run the original deep_sort code(using the detections provided) and this code(deep_sort_yolov3) on the test video (MOT16-06.mp4), the former's accuracy  is higher, is it the problem of detection phase?  Or the original deep_sort use the MOT16 training dataset to train? They both use the mars-small128 pretrained model after all. thanks in advance……

**Comment by Qidian213** [comment]
For the bug i forgot to update on github `if not track.is_confirmed() or track.time_since_update > 1: ` demo.py . You can try again .Actually, detections can affect it's performance . because use the same model and algorithm you should got almost the same accuracy.
